README.md

```diff
@@ -189,10 +189,16 @@ determining that location is as follows:
 -   [gcloud sdk](https://cloud.google.com/sdk/install) >= 269.0.0
 -   [jq](https://stedolan.github.io/jq/) >= 1.6
 -   [Terraform](https://www.terraform.io/downloads.html) >= 0.13.0
--   [terraform-provider-google] plugin >= 3.1, < 4.0
--   [terraform-provider-google-beta] plugin >= 3.1, < 4.0
+-   [terraform-provider-google] plugin ~> 4.5
+-   [terraform-provider-google-beta] plugin ~> 4.5
 -   [terraform-provider-gsuite] plugin 0.1.x if GSuite functionality is desired
 
+#### `terraform-provider-google` version 3.x
```

Comment:
I think this is unnecessary since we have upgrade guides and release notes now. We can remove this and the one below it.

Reply:
Sure - Would you like a new upgrading_to_project_factory_v12.0.md?  Or do you think the existing v12 release notes are sufficient?

Reply:
@apeabody I think we can skip v12 since release is out anyway but could you append to the v13 release notes that some submodules like `modules/project_services` now require 4.5 since we are unifying them. Some submodules like `project_services` tend to get used standalone

Reply:
Hi @bharathkkb for simplicity I've limited this to allowing TPG 4.0 on the remaining tests and updating everything to the TF 0.13+ format.  That way will not impact the minimum version individually of any of the submodules.